src/backend/PortabilityService.ConfigurationSource/ConfigurationSource.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Extensions.Configuration.Memory;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace PortabilityService.ConfigurationSource
+{
+    public class PortabilityServiceConfigurationSource : MemoryConfigurationSource
+    {
+        private const string DefaultConfigurationServiceUrl = "http://localhost:1443";
+        internal const string PortabilityServiceConfigurationRoot = "PortabilityServiceSettings";
+
+        public PortabilityServiceConfigurationSource(string configurationSection = PortabilityServiceConfigurationRoot)
```

Comment:
I would remove this override. I'd rather people have to set an environment variable in the override below.

Reply:
I didn't do it this way because I didn't want null passed in for the URI

Reply:
Sounds good.